modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/SleepTimer.kt

```diff
@@ -43,6 +56,23 @@ class SleepTimer @Inject constructor(@ApplicationContext private val context: Co
         }
         createAlarm(time.timeInMillis)
     }
+    suspend fun restartSleepTimerIfApplies(currentEpisodeUuid: String, isSleepTimerRunning: Boolean, onRestartSleepAfterTime: () -> Unit, onRestartSleepOnEpisodeEnd: suspend () -> Unit) {
```

Comment:
Unless I'm missing something I don't think that this function and `onRestartSleepOnEpisodeEnd` parameter need to be suspending.

Reply:
No need to use `suspend` anymore. When I was creating this function, in one of the implementations options, I have to use `suspend`. Nice catch! 

I will remove it